### ISSUE TITLE
Adjust 'Release Guidelines Checks' dashboard for TRG 4.02

### ISSUE THREAD

**Issue by evegufy** [eclipse-tractusx/sig-infra#259]
**Is your support request related to a problem? Please describe.**  

on the ['Release Guidelines Checks' dashboard](https://eclipse-tractusx.github.io/sig-release/), portal(backend) is shown as apparently not compliant with TRG 4.02 but portal-backend is actually compliant with that TRG, see:
https://github.com/eclipse-tractusx/portal-backend/tree/main/docker
https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02#description

also for cx-iam the dashboard shows non compliance with [TRG 4.02.](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02): I opened https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/383 for that

**Describe the solution you'd like**  
Adjust dashboard to show portal and cx-iam compliant with [TRG 4.02.](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02)

**Comment by FaGru3n** [comment]
Hi @evegufy, your PR was approved by @SebastianBezold, think your waiting for feedback from @AngelikaWittek .

For the `Release Guidelines Checks` we have to check if the result for desired systems.

**Comment by FaGru3n** [comment]
Nightly Run worked: 
https://github.com/eclipse-tractusx/sig-release/actions/runs/6307042100

**Comment by evegufy** [comment]
@carslen I'm reopening the issue because the portal(backend) and cx iam shown are still shown as not compliant with TRG 4.02 on the ['Release Guidelines Checks' dashboard](https://eclipse-tractusx.github.io/sig-release/),

**Comment by FaGru3n** [comment]
little update for cx-iam
![image](https://github.com/eclipse-tractusx/sig-infra/assets/121097161/7501226e-c2a7-4fc9-aa4f-b9ac96fa17c4)

but portal-backend seems working differently 
![image](https://github.com/eclipse-tractusx/sig-infra/assets/121097161/20659afd-143e-4de4-9129-5e41b315d019)

**Comment by tomaszbarwicki** [comment]
@FaGru3n, @evegufy  yes, that's known issue at the moment we don't support image aliases in Dockerfiles at our check. I've just created an issue to fix that in nearest future - https://github.com/eclipse-tractusx/tractusx-quality-checks/issues/53

cc: @SebastianBezold

**Comment by hzierer** [comment]
moved to our backlog to plan it properly
